### PR TITLE
#4015 Noti for new version is ready

### LINKED ISSUE #4015: Update new version toast style
I guess this is the perfect opportunity to style a new update toast in the same neo-brutalism style. 
Now I've realized we don't have it in design and @exezbcz can sketch something! :) 

image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #4015
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-04-02 at 4 21 09 PM

### CODE REVIEW

**Comment by prachi00** [comment]
After a lot of experiments with lots of components, this was the best I could come up for this
PS Wasn't able to add 3 lines on the left side as we can only have one `:before` element 

One catch: since the css is global, it would get applied for all dialogs if we add any in future, right now we are not using dialog component so it is okay and in future if we have to, maybe we use oruga dialog instead of buefy? 

@roiLeo @exezbcz

**Comment by prachi00** [comment]
or maybe we can use oruga modal here https://oruga.io/components/modal.html
it supports giving a class prop
what do you think?

**Review by roiLeo — CHANGES_REQUESTED** [review]
Please create NeoComponent with Oruga-ui as we're droping buefy

File: plugins/pwa.ts [hunk 1]
```diff
@@ -16,19 +16,15 @@ export default async () => {
           'For more details, visit https://pwa.nuxtjs.org/'
       )
 
-      if (event.isUpdate) {
+      if (event.isUpdate || true) {
```
Comment (daiagi) [inline plugins/pwa.ts:19]: Why ll true?

File: styles/global.scss [hunk 2]
```diff
@@ -371,3 +371,75 @@ hr {
     }
   }
 }
+
+.dialog {
```
Comment (daiagi) [inline styles/global.scss:375]: can this css be contained inside a component?

Comment (prachi00) [reply styles/global.scss:375]: this component is changing, I'll change this pr to draft for now

**Comment by prachi00** [comment]
going to close this one for now
Not able to figure out how to use oruga in pwa.ts file, will raise a new pr later

**Pull request closed without merging**